api_backend.yaml

```diff
@@ -307,6 +307,45 @@ paths:
           description: Profile cannot be updated.
           schema:
             $ref: "#/definitions/ProblemJson"
+  "/api-profile":
+    x-swagger-router-controller: ProfileController
+    get:
+      operationId: getApiUserProfile
+      summary: Get user's profile stored into the API
+      description: Returns the profile for the user identified by the provided fiscal code.
+      responses:
+        "200":
+          description: Found.
+          schema:
+            $ref: "#/definitions/Profile"
```

Comment:
io-functions-app always return ExtendedProfile

Reply:
Fixed in 9fa999b